docs/blog/2023-09-06-distributed-key-generation.md

```diff
@@ -0,0 +1,154 @@
+
+😃 🏊🏾‍♂️ ****1991: Pedersen**** 
+
+In the early 90s, Pedersen initially introduced the DKG scheme. The security in the initial scheme lies in the security of the Joint-Feldman Protocol for verifiable secret sharing during the secret-sharing process
+
+🙄⚓**********2006: [Gennaro et al.](https://link.springer.com/article/10.1007/s00145-006-0347-3)** 
+
+Gennaro et al. took the Joint-Feldman protocol as described by Pedersen and they showed that a potential adversary can influence the contribution of the public key in a way that is not being uniformly chosen randomly. Therefore, the adversary can take the advantage to compromise two of the participants and gain information about the private key, which means it leaks information about the secret. As a result, this is a sinking phase for Pedersen, as it shows that it cannot be used as a secure DKG protocol.
+
+😃 🏊🏾‍♂️ **2007**:  **[Rabin et al.](https://link.springer.com/content/pdf/10.1007/3-540-48910-X_21.pdf)** 
+
+In this 2007 study, Rabin et al. provided a series of security proofs showing that Feldman's (VSS) was susceptible to malicious inputs in Pedersen's (DKG). However, the same article also notes in Section 5 that Joint-Feldman (JF-DKG) has a beneficial attribute: it prevents any adversary from producing an output that allows the computation of its discrete logarithm. This feature is sufficiently robust to enable the construction of secure threshold signature schemes. Therefore, it makes Pedersen’s initial DKG sufficient for threshold signing. 
+
+😃 🏊🏾‍♂️ **************2021: [Gurkan et al](https://eprint.iacr.org/2021/005).**
+
+In a more recent study, Gurkan et al. showed that Pedersen's DKG is a secure way to handle rekeyable encryption schemes, signature schemes, and VUF schemes. Specifically, they provided proof that using JF-DKG is secure, preventing an adversary from forging a signature by participating in the actual DKG protocol.
+
+# ***The Future: Asynchronous DKG Algorithms***
+
+Looking ahead, one exciting development in the world of DKG is the adoption of asynchronous algorithms. Traditional DKG protocols often rely on synchronous communication assumptions, where participants have to wait for messages from others before proceeding.
+
+The adoption of asynchronous DKG algorithms aims to enhance the efficiency and resilience of decentralized systems, such as blockchain networks, which are becoming increasingly popular. By improving the underlying key generation processes, these algorithms are paving the way for more secure, reliable, and scalable decentralized systems.
+
+The latest advanced DKG protocol presented is [Bingo](https://eprint.iacr.org/2022/1759.pdf) 🀞🀞:
+
+- Interesting components adjusted in Bingo:
+    - [Bivariate Polynomials](https://mathworld.wolfram.com/BivariatePolynomial.html)
```

Comment:
That would be nice to explain what is the advantage of using bivariate, what does it bring to the table (IIRC it's for the adaptive security ?)